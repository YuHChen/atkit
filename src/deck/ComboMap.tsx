import React from "react";

import Wip from "../Wip";

const ComboMap = () => <Wip className="mt-3" pageName="deck combo map page" />;

export default ComboMap;
