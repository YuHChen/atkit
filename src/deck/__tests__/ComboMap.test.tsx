import React from "react";
import renderer from "react-test-renderer";

import ComboMap from "../ComboMap";

describe("ComboMap", () => {
  test("renders as expected", () => {
    const tree = renderer.create(<ComboMap />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
