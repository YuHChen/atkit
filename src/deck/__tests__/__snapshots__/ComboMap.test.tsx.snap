// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComboMap renders as expected 1`] = `
<div
  className="fade mt-3 alert alert-info show"
  role="alert"
>
  <div
    className="alert-heading h4"
  >
    Pardon the mess
  </div>
  <p
    className="mb-0"
  >
    This 
    deck combo map page
     is still a work in progress.
  </p>
</div>
`;
